### PR TITLE
Fix interpolation of opd when rotating

### PR DESCRIPTION
**Pull request by gmegh**

### CODE REVIEW

**Review by jbkalmbach — CHANGES_REQUESTED** [review]
Just a couple of quick suggestions on the code itself. But I think we should add a short test for this function making sure the if we rotate the OPD we get rotated Zernikes like we expect.

**Comment by gmegh** [comment]
Hi @jbkalmbach ! Please, have a look at the suggested unit test. Once you give your approval I will squash the commits and merge it! Thanks

File: python/lsst/ts/imsim/imsimCmpt.py [hunk 1]
```diff
@@ -564,10 +564,11 @@ def _mapOpdToZk(self, rotOpdInDeg, numOpd):
 
             # Rotate OPD if needed
             if rotOpdInDeg != 0:
-                opdRot = opd
+                opdRot = opd.copy()
+                minValue = np.nanmin(np.abs(opdRot))
```
Comment (jbkalmbach) [inline python/lsst/ts/imsim/imsimCmpt.py:568]: Could you just add a comment here explaining the need for the minValue parameter?

**Review by jbkalmbach — APPROVED** [review]
Looks good. Thanks!

File: python/lsst/ts/imsim/imsimCmpt.py (hunk 1 above)
Comment (gmegh) [reply python/lsst/ts/imsim/imsimCmpt.py:568]: Sorry I missed this! Just added the comment

**Pull request merged**